modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
+
+[[]]
+==== Hindi, Malay, and Vietnamese translations added to four Premium Plugins
+
+The following four Premium Plugins now present in Hindi, Malay, and Vietnamese:
+
+* Accessibility Checker
+* AdvTable
+* Comments
+* Spellchecker
+
+
+[[changed-plugins]]
+=== Changed plugins
+
+[[autoresize]]
+==== Autoresize
+
+Previously, the `autoresize` plugin offered an `autoresize_on_init` option.
+
+This setting does not do anything meaningful, and had been previously deprecated.
+
+As of {productname} 6.0, `autoresize_on_init` has been removed from the `autoresize` plugin.
+
+[[media]]
+==== Media
+
+In {productname} 5.x, the `media` plugin used https://tiny.cloud/docs/api/tinymce.html/tinymce.html.saxparser/[SaxParser] to validate elements for parsing.
+
+As of {productname} 6.0, SaxParser is no longer used. Another {productname} public API — DomParser — is used instead.
+
+As a consequence the `iframe`, `video`, `audio`, and `object` elements are no longer marked as _special_. Instead their contents are simply validated against the {productname} schema.
+
+NOTE: The {productname} public API, DomParser uses, in turn, the same-named browser parsing API: https://developer.mozilla.org/en-US/docs/Web/API/DOMParser[DomParser].
+
+Also, the `media` plugin’s `media_scripts` option was deprecated in {productname} 5.10.
+
+// Based on the code it looks like `media_scripts` was meant to be a way to provide an allowed list of <script>s that could be used to embed media. However, somewhere along the lines it broke and no longer works.
+
+As of the release of {productname} 6.0, this option has been removed completely.
+
+[[noneditable]]
+==== Noneditable
+
+The `noneditable` plugin is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `noneditable` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[paste]]
+==== Paste
+
+The `paste` plugin’s functionality is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `paste` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+The `paste_data_images` option now defaults to `true`. When `paste` was a plugin, this option was, by default, set to `false`, which prevented images being pasted from the local machine.
+
+The `mceInsertClipboardContent` argument, `content`, has been renamed. It is now `html`.
+
+The new name is a more accurate reflection of what sort of data the argument passes.
+
+NOTE: `PastePreProcess` also takes an argument, `content`. This argument, for this command, has not changed.
```

Comment:
```suggestion
```
This isn't correct, as it's an event so it can't take arguments. I think we can just remove this line.